vertical-pod-autoscaler/README.md

```diff
@@ -7,6 +7,7 @@
 - [Getting started](#getting-started)
 - [Components and Architecture](#components-and-architecture)
 - [Features and Known limitations](#features-and-known-limitations)
+- [Recommender metrics](#recommender-metrics)
```

Comment:
Could this be just "Metrics" ?
```suggestion
- [Metrics](#Metrics)
```

Since there are 3 components, and in the linked document, we can make a header for each component

Reply:
Thank you for you review. I will fix this